pages/docs/couple-your-code/couple-your-code-time-step-sizes.md

```diff
@@ -75,6 +75,45 @@ This procedure is independent of whether a serial or a parallel coupling scheme
 For parallel coupling, both solvers run together and everything happens simultaneously in both participants, while for serial coupling, the first participant needs reach the end of the window before the second one can start.
 {% endnote %}
 
+### Possible subcycling pitfall
+
+If you are using very small many time steps in one window, you might see the following error message:
+```
+preCICE has detected a difference between its internal time and the time of this participant. This can happen, if you are using very many substeps per time window over multiple time windows.
```

Comment:
```suggestion
ERROR: preCICE has detected a difference between its internal time and the time of this participant. This can happen, if you are using very many substeps per time window over multiple time windows.
```